SophiaS/index.html

```diff
@@ -16,4 +16,5 @@ <h1>
     <h2>
         Hello everyone, this is Sophia, I like go to shopping and find popular styles each year.
     </h2>
+    </h3> I want to create a new branch.
```

Comment:
@Sophiashe88 h3 element has to have an opening tag `<h3>` and a closing tag `</h3>`, please add it.

Reply:
Thank you!